apps/mobile/src/features/ImportReadOnly/ImportAccountForm.container.tsx

```diff
@@ -0,0 +1,165 @@
+import { View, Text, ScrollView } from 'tamagui'
+import { useLocalSearchParams } from 'expo-router'
+import { LargeHeaderTitle } from '@/src/components/Title/LargeHeaderTitle'
+import { useScrollableHeader } from '@/src/navigation/useScrollableHeader'
+import { NavBarTitle } from '@/src/components/Title'
+import React, { useState, useEffect, useCallback } from 'react'
+import { XStack, YStack } from 'tamagui'
+import { SafeButton } from '@/src/components/SafeButton'
+import { KeyboardAvoidingView, StyleSheet } from 'react-native'
+import { SafeAreaView, useSafeAreaInsets } from 'react-native-safe-area-context'
+import { SafeOverview } from '@safe-global/store/gateway/AUTO_GENERATED/safes'
+import { makeSafeId } from '@/src/utils/formatters'
+import { useAppSelector } from '@/src/store/hooks'
+import { selectAllChainsIds } from '@/src/store/chains'
+import { useLazySafesGetOverviewForManyQuery } from '@safe-global/store/gateway/safes'
+import { NetworkBadgeContainer } from '@/src/features/ImportReadOnly/NetworkBadge.container'
+import { useRouter } from 'expo-router'
+import { SafeInput } from '@/src/components/SafeInput/SafeInput'
+import { Identicon } from '@/src/components/Identicon'
+import { SafeFontIcon } from '@/src/components/SafeFontIcon'
+import { isValidAddress } from '@safe-global/utils/validation'
+import { parsePrefixedAddress } from '@safe-global/utils/addresses'
+import { CircleSnail } from 'react-native-progress'
+
+const AvailableNetworks = ({ networks }: { networks: SafeOverview[] }) => {
+  return (
+    <YStack marginTop={'$5'} gap={'$1'}>
+      <Text fontWeight={'600'}>Available on networks:</Text>
+      <XStack gap={'$1'} marginTop={'$3'}>
+        {networks?.map((safe) => <NetworkBadgeContainer key={safe.chainId} chainId={safe.chainId} />)}
+      </XStack>
+    </YStack>
+  )
+}
```

Comment:
shall we move it to be inside `ImportAccountForm/components/AvailableNetworks` folder?